### PR TITLE
Forego syntax highlighting on code within lists

### PR DESCRIPTION
**Pull request by cbandy**
Older versions of Hugo do not handle it well. This section has been rearranged in later versions of the docs, avoiding the issue entirely.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)

**What is the current behavior? (link to any open issues here)**

Screen Shot 2020-07-15 at 2 13 07 PM

**What is the new behavior (if this is a feature change)?**

Screen Shot 2020-07-15 at 2 14 39 PM

### CODE REVIEW

File: docs/content/architecture/namespace.md [hunk 1]
```diff
@@ -144,51 +144,49 @@ As described in the Namespace Operating Mode section above, when using either th
...
-    subjects:
-    - kind: ServiceAccount
-      name: postgres-operator
-      namespace: <postgresql-operator-namespace>
-    ```
+    within the namespace to create PostgreSQL clusters.  This is specifically be done by creating the following
+    Role and RoleBinding within the target namespace:
+
+        ---
+        kind: Role
```
Comment (jkatz) [inline docs/content/architecture/namespace.md:151]: The convention we've been doing is

```
---
kind: Role
...
```

Comment (cbandy) [reply docs/content/architecture/namespace.md:151]: The triple-ticks? That's what breaks in older Hugo.

Comment (jkatz) [reply docs/content/architecture/namespace.md:151]: Even without `yaml`?

Comment (jkatz) [reply docs/content/architecture/namespace.md:151]: Fine, let's just get it in.

Comment (cbandy) [reply docs/content/architecture/namespace.md:151]: Yep. Any fenced block was failing when indented. The solution here is to fallback to indent-only formatting.

**Pull request merged**